### PR TITLE
404 + Cleanup

### PR DESCRIPTION
**Pull request by cmc333333**
- Custom 404
- Removes blocks which are no longer needed; replaces with "title" context in all views
- Removes paths + templates which are no longer used
- Removes #amount-form from links on the landing/sorter pages

### CODE REVIEW

File: peacecorps/peacecorps/templates/404.jinja [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% extends "donations/base.jinja" %}
+
+{% block content %}
+<section class="blurb blurb--sm">
+  <div class="t-blurb--md t--black">Error</div>
```
Comment (msecret) [inline peacecorps/peacecorps/templates/404.jinja]: This should probably be an h1 rather then a div.

File: peacecorps/peacecorps/templates/404.jinja [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{% extends "donations/base.jinja" %}
+
+{% block content %}
+<section class="blurb blurb--sm">
```
Comment (msecret) [inline peacecorps/peacecorps/templates/404.jinja]: If you put the t--black class here, you won't have to repeat it twice below ( I think).

File: peacecorps/peacecorps/templates/404.jinja [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  <div class="t-blurb--md t--black">Error</div>
+  <div class="t--black">
```
Comment (msecret) [inline peacecorps/peacecorps/templates/404.jinja]: This could be a p or a h4.

**Comment by cmc333333** [comment]
Change to h1/h4. Forgot a screenshot the first time. Note that this is the 404 page for donations alone, so we're using the donations header/footer.

screen shot 2015-01-16 at 9 38 32 am

**Pull request merged**